js/stringProcessing/findKeywordInUrl.js

```diff
@@ -13,7 +15,7 @@ module.exports = function( url, keyword ) {
 
 	if ( formatUrl !== null ) {
 		formatUrl = formatUrl[ 0 ].replace( /<.*?>\s?/ig, "" );
-		if ( formatUrl.match( keywordRegex( keyword ) ) !== null ) {
+		if ( matchTextWithTransliteration( formatUrl, keyword ).length > 0 ) {
```

Comment:
I would return `( matchTextWithTransliteration( formatUrl, keyword ).length > 0 )` directly instead of doing and instead of `return keywordFound` in the end of the function just `return false`;